src/services/ann-index.ts

```
@@ -898,13 +1305,288 @@
+    queryCount: number = 100
+  ): Promise<{
+    buildTime: number;
+    searchTime: number;
+    memoryUsage: number;
+    accuracy: number;
+  }> {
+    const config = { ...BENCHMARK_ANN_CONFIG, algorithm: indexType };
+    const index = createANNIndex(config);
+    
+    // Generate test data
+    const vectors = this.generateTestVectors(vectorCount, dimension);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)